gremlin-server/pom.xml

```diff
@@ -63,10 +63,12 @@ limitations under the License.
         <dependency>
             <groupId>com.codahale.metrics</groupId>
             <artifactId>metrics-graphite</artifactId>
+            <optional>true</optional>
```

Comment:
Shouldn't this include the version? Asking because you've explicitly mentioned 3.0.2 before.